rs_msgs/msg/GaitInput.msg

```diff
@@ -0,0 +1,13 @@
+float64 x 
+float64 y 
+float64 z 
+float64 roll 
+float64 pitch 
+float64 yaw 
+float64 StepLength 
+float64 LateralFraction 
+float64 YawRate 
+float64 StepVelocity 
+float64 ClearanceHeight 
+float64 PenetrationDepth 
+float64 SwingPeriod 
```

Comment:
I believe that using "float" instead of float 64 should be enough in this case

Reply:
resolved in f7e5eb4c11d45bbf767f8d51dc04b079b425596a